templates/boost_development.html

```diff
@@ -0,0 +1,31 @@
+{% extends "base.html" %}
+
+{% block title %}Boost Development{% endblock %}
+
+{% block content_wrapper %}
+<div class="flex justify-center items-center my-4">
+    <h1 class="text-2xl">Development</h1>
+</div>
+<div class="container px-4 mb-8 mx-auto">
```

Comment:
can we add the 'card' class here to wrap the top part in a card, and have less of the grey background?